incubator/kubeless/templates/kafka-headless-svc.yaml

```diff
@@ -0,0 +1,14 @@
+# A headless service to create DNS records
+apiVersion: v1
+kind: Service
+metadata:
+  name: broker
+  namespace: kubeless
+  labels:
```

Comment:
Label `kubeless` (or rather `component`?) should be added here.

Reply:
added standard labels